recipes/mtspec/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "0.3.0" %}
+{% set sha256 = "307dc34acefdfae06f055d99d0260cb3529a1fd46d7696be2ab533752aff01a1" %}
+
+package:
+  name: mtspec
+  version: {{ version }}
+
+
+source:
+  fn: mtspec-{{ version }}.zip
+  url: https://pypi.io/packages/source/m/mtspec/mtspec-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - gcc
+    - numpy
```

Comment:
Do you use `numpy`. include in `setup.py`? If so we need to change here (and below) to `numpy x.x` to ensure the same `numpy` used to build will be installed at run time by baking the `numpy` build on the package.